homeassistant/components/sensor/discogs.py

```diff
@@ -25,47 +27,85 @@
 
 DEFAULT_NAME = 'Discogs'
 
-ICON = 'mdi:album'
-
-SCAN_INTERVAL = timedelta(hours=2)
+ICON_RECORD = 'mdi:album'
+ICON_PLAYER = 'mdi:record-player'
+UNIT_RECORDS = 'records'
+
+SCAN_INTERVAL = timedelta(minutes=10)
+
+SENSOR_COLLECTION_TYPE = 'collection'
+SENSOR_WANTLIST_TYPE = 'wantlist'
+SENSOR_RANDOM_RECORD_TYPE = 'random_record'
+
+SENSORS = {
+    [SENSOR_COLLECTION_TYPE]: {
```

Comment:
The keys shouldn't be lists.

Reply:
Sorry, I wasn't thinking and using the JS method of computed keys